packages/0xcert-ethereum-http-provider/src/core/provider.ts

```diff
@@ -111,9 +111,8 @@ export class HttpProvider extends GenericProvider {
    * Sends the RPC call.
    */
   public send(data: any, callback: (err, data) => any) {
-
     const { url, ...options } = {
-      url: 'http://localhost:8545',
```

Comment:
Why was the default port changed to 8524? It should probably stay as before.

Reply:
8524 is the port used by specron

Reply:
Yes but geth and parity use `8545` port as default. So default port should not be set to test environment but where it will actually gets used.

Reply:
my mistake, noticed I changed both ports